### ISSUE TITLE
Service stoped working after several hours

### ISSUE THREAD

**Issue by robspaltenstein** [romanlum/LoxBerry-Plugin-Zigbee2Mqtt#11]
I installed the Gateway on a Loxberry with other plugins installed.
After several hours the service stoped working. When I restart the service with save the configuration again, then the devices are again recognised and it works at least several hours.
I installed the plugins on a clean Raspberry with the newest loxberry version. The same behaivor occured also here.

Configuration:
* Raspberry PI 3
* Loxberry 2.0.1.3
* MQTT Gateway 1.1
* Zigbee2Mqtt 0.3
* ConBee2
* Aqara Wireless Remote Switch
* Ikea Kadrilj Rollo with Tradfri wireless remote switch

**Comment by romanlum** [comment]
Thanks for the report.
Is it possible to attach the logfile of the service?

**Comment by robspaltenstein** [comment]
Here are the logs for both services:
zigbee2mqtt.log and mgttgateway.log

12:02 Restart the loxberry
> the "Schalter-001" is working fine
12:07 turn on "Add new devices" and save this setting
12:35 New "Schalter-002" added, change the friendly name
12:27 All switches works fine
12:45 new Test on both switches - nothing works
13:15 new Test on both switches - nothing works
14:35 new Test on both switches - nothing works
14:36 Restart Service
14:37 All switches works fine
14:50 new Test on both switches - nothing works
[zigbee2mqtt.log](https://github.com/romanlum/LoxBerry-Plugin-Zigbee2Mqtt/files/4977638/zigbee2mqtt.log)
[mqttgateway.log](https://github.com/romanlum/LoxBerry-Plugin-Zigbee2Mqtt/files/4977639/mqttgateway.log)

Many thanks for jour support
Robert

**Comment by robspaltenstein** [comment]
I tried to update the Raspberry. The Loxberry softtware is on the current version 2.0.1.3. I use the direct commands:
* su -  (to switch to root rights)
* apt update
* apt full-upgrade

I did not know where I can find the update logs, but I think there was an update on php an npm.
After the restart of the Loxberry i runs 5 hours.
Yesterday it runs the whole day, but after a restart of the service this morning it runs only about two hours.
I attached the new logs.
Robert
[zigbee2mqtt (1).log](https://github.com/romanlum/LoxBerry-Plugin-Zigbee2Mqtt/files/5000094/zigbee2mqtt.1.log)
[mqttgateway (1).log](https://github.com/romanlum/LoxBerry-Plugin-Zigbee2Mqtt/files/5000096/mqttgateway.1.log)

**Comment by romanlum** [comment]
Thanks for the logs.
I cannot find any particular problem of the plugin....

This may is a problem on zigbee2mqtt, because the plugin is a wrapper around it.

I will create a new version of the plugin that includes the latest version of zigbee2qtt.

**Comment by romanlum** [comment]
Pre-Release created.
Please install the prerelease version 0.4.0 and check if the problem is gone.

**Comment by robspaltenstein** [comment]
It's now working!
I installed the Loxberry new from scratch. I set the language to English, not to German. Did all the updates and installed the MQTT and the zigbee2mqtt plugins.
In the meantime I installed the deConz Software on a Windows PC and there was an update for the ConBee2 Firmware.
From:
Coordinator firmware version: '{"type":"ConBee2","meta":{"transportrev":0,"product":0,"majorrel":38,"minorrel":74,"maintrel":0,"revision":"0x264a0700"}}'
To New:
Coordinator firmware version: '{"type":"ConBee2","meta":{"transportrev":0,"product":0,"majorrel":38,"minorrel":88,"maintrel":0,"revision":"0x26580700"}}'

I leave the Port Setting blank and I could pair several devices. I pair a "Temperature and Humidity Sensor" from Aqara and it send all data changes to the gateway.
I also paired an IKEA TRADFRI Smart Plug. It shows me that an update is availlable. With the old version the upgrade does not work - with the new version I was able to upgrade the device:
http://loxberry:PASS@192.168.111.232:80/admin/plugins/mqttgateway/mqtt.php?topic=zigbee2mqtt/bridge/ota_update/update&value=steckdose001

Thank you for your support and for this great plugin.
Robert
